context/app/static/js/components/entity-search/facets/Facets/hooks.js

```diff
@@ -4,7 +4,12 @@ function useGroupedFacets(resultsFacets) {
   const { facets } = useStore();
 
   return resultsFacets.reduce((acc, facet) => {
+    if (!(facet.identifier in facets)) {
+      return acc;
+    }
```

Comment:
Is it worth a comment explaining why/when the facet identifier would be missing from the list?

Reply:
This is only an issue because we don't have a loading state. The facets configuration updates before the api hits. We could reduce over the configured facets instead of the results facets, but lookup to get the correct facet in results facets is more convoluted.